content/docs/relicensing.md

```diff
@@ -60,19 +60,22 @@ We also have worked with our legal counsel to build several more boring pieces:
 * A corporate agreement to relicense that is available for companies to sign
   and has begun to be distributed to some of the known and/or large contributors
 
-We currently plan to install the new developer policy and add the new license in
-January 2019 after the LLVM 8.0 release has branched.
+We have installed the new developer policy and added the new license on January
+19th 2019 after the LLVM 8.0 release had branched.
 
-Once that is done, we will still need need to:
+We are currently collecting relicensing agreements from copyright holders of
+contributions before 19th of January 2019. While we already have well over 90%
+of code covered, we are still working through the long tail. Help is very much
+appreciated with that. Please have a look at our
```

Comment:
I would change this to say "We are seeking volunteers to help us reach individuals and corporations who have not signed the relicense agreements. For full details on how to help, please see our ...

Reply:
Thanks @tlattner! I used your suggested text in the latest update.